configs/recognition/c2d/README.md

```diff
@@ -0,0 +1,93 @@
+# C2D
+
+<!-- [ALGORITHM] -->
+
+C2D is the baseline of [Non-local Neural Networks](https://arxiv.org/abs/1711.07971)
+
+NOTICE: C2D implementations are slightly different between 1.The paper above; 2."SlowFast" repo; 3."Video-Nonlocal-Net" repo.
+
+C2D implementation in MMAction2 is kept same as the ["Video-Nonlocal-Net" repo](https://github.com/facebookresearch/video-nonlocal-net/tree/main/scripts/run_c2d_baseline_400k.sh)
+
+Specifically:
+
+- maxpool3d_1 = nn.MaxPool3d(kernel_size=(1, 3, 3), stride=(1, 2, 2), padding=(0, 0, 0))
+- maxpool3d_2 = nn.MaxPool3d(kernel_size=(2, 1, 1), stride=(2, 1, 1), padding=(0, 0, 0))
+
+C2D_Nopool implementation in MMAction2 is kept same as the ["SlowFast" repo](https://github.com/facebookresearch/SlowFast/blob/main/configs/Kinetics/c2/C2D_NOPOOL_8x8_R50.yaml)
+
+<!-- [ABSTRACT] -->
+
+<!-- [IMAGE] -->
+
+## Results and Models
+
+### Kinetics-400
+
+| frame sampling strategy | scheduler |   resolution   | gpus |   backbone    | pretrain | top1 acc | top5 acc |   ref top1 acc    |   ref top5 acc    | testing protocol  | inference time(video/s) | gpu_mem(M) |   config    |   ckpt    |    log    |
+| :---------------------: | :-------: | :------------: | :--: | :-----------: | :------: | :------: | :------: | :---------------: | :---------------: | :---------------: | :---------------------: | :--------: | :---------: | :-------: | :-------: |
+|          8x8x1          | MultiStep | short-side 320 |  8   | ResNet50 (original) | ImageNet |  73.16   |  90.88   | [67.2](https://github.com/facebookresearch/SlowFast/blob/main/MODEL_ZOO.md#kinetics-400-and-600) | [87.8](https://github.com/facebookresearch/SlowFast/blob/main/MODEL_ZOO.md#kinetics-400-and-600) | 10 clips x 3 crop |            x            |   21547    | [config](/configs/recognition/c2d/c2d_nopool_imagenet-pretrained-r50_8xb32-8x8x1-100e_kinetics400-rgb.py) | [ckpt TODO](TODO) | [log TODO](TODO) |
+|          8x8x1          | MultiStep | short-side 320 |  8   | ResNet50 (with temporal pooling) | ImageNet |  71.78   |  89.90   | [71.9](https://github.com/facebookresearch/video-nonlocal-net#modifications-for-improving-speed) | [90.0](https://github.com/facebookresearch/video-nonlocal-net#modifications-for-improving-speed) | 10 clips x 3 crop |            x            |   17006    | [config](/configs/recognition/c2d/c2d_imagenet-pretrained-r50_8xb32-8x8x1-100e_kinetics400-rgb.py) | [ckpt TODO](TODO) | [log TODO](TODO) |
+
+1. The values in columns named after "reference" are the results reported in the original repo, using the same model settings.
+
+2. The **gpus** indicates the number of gpu we used to get the checkpoint. It is noteworthy that the configs we provide are used for 8 gpus as default.
```

Comment:
replace this note with: The **gpus** indicates the number of gpus we used to get the checkpoint. If you want to use a different number of gpus or videos per gpu, the best way is to set `--auto-scale-lr` when calling `tools/train.py`, this parameter will auto-scale the learning rate according to the actual batch size and the original batch size. Refer to [swin](https://github.com/open-mmlab/mmaction2/pull/1939/files#diff-1c10da6b25716e7db3b38da277f02d45dec701d289c48681cb8b4230bf3454b9).